epochStart/metachain/economics.go

```diff
@@ -244,3 +256,18 @@ func (e *economics) VerifyRewardsPerBlock(
 func (e *economics) IsInterfaceNil() bool {
 	return e == nil
 }
+
+func logEconomicsDifferences(computed *block.Economics, received *block.Economics) {
+	log.Debug("VerifyRewardsPerBlock error",
```

Comment:
log.Warn and make computed and received in separate lines. it would be easier to read.

Reply:
done